truss/tests/test_model_inference.py

```diff
@@ -454,6 +454,54 @@ def predict(self, request):
         assert "levelname" in relevant_line
 
 
+@pytest.mark.integration
+def test_multiple_request_ids_logged():
+    class Model:
+        def predict(self, request):
+            print("This is a message from the Truss: Hello World!")
+            return {}
+
+    # TODO: REMOVE LOGURU HERE!!!
+    config = """model_name: printing-truss\nrequirements: [loguru]
```

Comment:
hacky workaround - this needs to be fixed

Reply:
what exactly is the issue here? why aren't the tests picking up your new requirements?

Reply:
I'm not sure why `loguru` isn't being picked up. I've added it to `pyproject.toml` as well as `server/requirements.txt` and `control/requirements.txt` (right under `python-json-logger`).